### DISCUSSION TITLE
Possibity of getting areas/properties of faces that are not the faces

### DISCUSSION THREAD

**Discussion by vishalnp37** [neperfepx/neper#728] [Q&A] [answered]
Hello, Is there a possibility to get ids and areas of the faces that are present on the inside of the tesselation?

**Comment by smallrain1996** [comment]
Without this function, the future seems impossible.

**Comment by rquey** [comment, marked as answer]
Use `-statface domtype,id,area` and consider only the faces with domtype = -1.
